bool override_status
bool pid_status
int16 rc_number
string flight_mode
